inst/extremes/extremes.Rmd

```diff
@@ -26,5 +26,8 @@ cat(getLogo())
 tbl <- extremesTable(data)
 # formTable <- padTable(tbl)
 # cat(formTable)
-kable(tbl, align='r', row.names=FALSE)
+if(!identical(tbl, "The dataset requested is empty.")){
```

Comment:
I would nest this boolean logic a function. Right now you are relying on two hardcoded strings from two different places to be equal. A dev may change it in one place but not the other.